lib/inc/internal/facts/linux/virtualization_resolver.hpp

```diff
@@ -34,7 +34,8 @@ namespace facter { namespace facts { namespace linux {
          * @param leases_file The location of where the leases file exists.
          * @return Returns "azure" if running on azure, otherwise an empty string.
          */
-        static std::string get_azure(collection& facts, std::string const& leases_file = "/var/lib/dhcp/dhclient.eth0.leases");
+        static std::string get_azure();
```

Comment:
can you update the documentation to reflect the updated `get_azure()` function, and the newly added `get_azure_from_leases_file` function

Reply:
don't we need the `collection& facts` param anymore?